Tests/DashboardPageE2ETests.cs

```diff
@@ -0,0 +1,394 @@
+            string actualMonthAndYear = wait.Until(webDriver => webDriver.FindElement(By.ClassName("H1MonthAndYear"))).Text;
+            
+            Assert.Equal(expectedMonthAndYear, actualMonthAndYear);
+            Teardown(driver);
+        }
+
+        [Fact]
+        public void DashboardPageShouldContainOneAdultSelectedDefault()
+        {
+            using IWebDriver driver = Setup();
+            WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(10));
+
+            string expectedNumberOfAdults = "1 adult";
+            string actualNumberOfAdults =
+                wait.Until(webDriver => webDriver.FindElement(By.Id("number-of-adults"))).GetDomAttribute("placeholder");
+            
+            Assert.Equal(expectedNumberOfAdults, actualNumberOfAdults);
+            Teardown(driver);
+        }
+        
+        [Fact]
+        public void DashboardPageShouldContainRoomTypesForOneTwoAndFourAdults()
+        {
+            using IWebDriver driver = Setup();
+            WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(10));
+            
+            string roomTypeOptions =
+                wait.Until(webDriver => webDriver.FindElement(By.Id("number-of-adults"))).Text;
+            bool containsValidOptions = roomTypeOptions.Split('\n').Length == 3 && 
+                                        roomTypeOptions.Contains("1 adult") && 
+                                        roomTypeOptions.Contains("2 adults") &&
+                                        roomTypeOptions.Contains("4 adults");
+            
+            Assert.True(containsValidOptions);
+            Teardown(driver);
+        }
+
+        [Fact]
+        public void DashboardPageShouldContainDefaultPredefinedListOfHotelsForFiltering()
+        {
+            using IWebDriver driver = Setup();
+            WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(10));
+            
+            wait.Until(webDriver => webDriver.FindElement(By.Id("selected-hotels-button"))).Click();
+            Thread.Sleep(1000); // wait to ensure all options are displayed
+            ReadOnlyCollection<IWebElement> hotelFilters =
+                wait.Until(webDriver => webDriver.FindElements(By.ClassName("HotelOptions")));
+            bool containsValidOptions = hotelFilters.All(options => 
+                options.Text is "All" or "Local" or "No budget" or "Cabinn Aalborg" or "Kompas Hotel Aalborg" or
+                    "Slotshotellet Aalborg" or "Aalborg Airport Hotel" or "Comwell Hvide Hus Aalborg" or
+                    "Helnan Phønix Hotel" or "Hotel Jomfru Ane" or "Hotel Scheelsminde" or "Milling Hotel Aalborg" or
+                    "Prinsen Hotel" or "Radisson Blu Limfjord Hotel Aalborg" or "Room Rent Prinsen" or
+                    "Scandic Aalborg City" or "Scandic Aalborg Øst" or "Zleep Hotel Aalborg");
+            bool isDistinctAndContainsAll = hotelFilters.Distinct().Count() == 18 && containsValidOptions;
```

Comment:
her kunne du istedet for at have en bool hvor der står And i, så kunne du i asserten bare bruge &&

Reply:
Det er same procedure. Så kan jeg bedre lide at kunne læse navnet på variablen i Assert, hvis man vil vide hvordan testen fungerer. Men begge dele ville være lige gode, det er bare preference.